src/canisters/frontend/ic-asset/src/sync.rs

```diff
@@ -306,6 +308,64 @@ pub(crate) fn gather_asset_descriptors(
                 warn!(logger, "{}", serde_json::to_string_pretty(&rule).unwrap());
             }
         }
+
+        let no_policy_assets = asset_descriptors
+            .values()
+            .filter(|asset| {
+                asset.config.warn_about_no_security_policy()
+                    && matches!(
+                        asset.config.security_policy,
+                        None | Some(SecurityPolicy::Disabled)
+                    )
+            })
+            .collect_vec();
+        if !no_policy_assets.is_empty() {
+            warn!(
+                logger,
+                "This project does not define a security policy for some assets."
+            );
+            warn!(
+                logger,
+                "You should define a security policy in .ic-assets.json5. For example:"
+            );
+            warn!(logger, "[");
+            warn!(logger, "  {{");
+            warn!(logger, r#"    "match": "**/*","#);
+            warn!(logger, r#"    "security_policy": "standard""#);
+            warn!(logger, "  }}");
+            warn!(logger, "]");
+
+            if no_policy_assets.len() == asset_descriptors.len() {
+                warn!(logger, "Assets without any security policy: all");
+            } else {
+                warn!(logger, "Assets without any security policy:");
+                for asset in &no_policy_assets {
+                    warn!(logger, "  - {}", asset.key);
+                }
+            }
+        }
+        let standard_policy_assets = asset_descriptors
+            .values()
+            .filter(|asset| {
+                asset.config.warn_about_standard_security_policy()
+                    && asset.config.security_policy == Some(SecurityPolicy::Standard)
+            })
+            .collect_vec();
+        if !standard_policy_assets.is_empty() {
+            warn!(logger, "This project uses the default security policy for some assets. While it is set up to work with many applications, it is recommended to further harden the policy to increase security against attacks like XSS.");
+            warn!(logger, "To get started, have a look at 'dfx info canister-security-policy'. It shows the default security policy along with suggestions on how to improve it.");
+            if standard_policy_assets.len() == asset_descriptors.len() {
+                warn!(logger, "Unhardened assets: all");
+            } else {
+                warn!(logger, "Unhardened assets:");
+                for asset in &standard_policy_assets {
+                    warn!(logger, "  - {}", asset.key);
+                }
+            }
+        }
+        if !standard_policy_assets.is_empty() || !no_policy_assets.is_empty() {
+            warn!(logger, "To disable the policy warning, define \"disable_standard_security_policy_warning\": true in .ic-assets.json5.");
```

Comment:
If I'm reading the config.rs right...
```suggestion
            warn!(logger, "To disable the policy warning, define \"disable_security_policy_warning\": true in .ic-assets.json5.");
```